### ISSUE TITLE
How to modify the value inside  TcpTransportConfiguration

### ISSUE THREAD

**Issue by cptjackwu** [apache/plc4x#1175]
### What would you like to happen?

  default boolean isKeepAlive() {
        return false;
    }

    default boolean isNoDelay() {
        return true;
    }

    default int getConnectTimeout() {
        return 1000;
    }

i want modify connectionTimeout  or isKeepAlive ,but PlcConnection   is no way for me to modify it。

PlcConnection plcConnection = new PlcDriverManager().getConnection(url);

### Programming Languages

- [X] plc4j
- [ ] plc4go
- [ ] plc4c
- [ ] plc4net

### Protocols

- [ ] AB-Ethernet
- [ ] ADS /AMS
- [ ] BACnet/IP
- [ ] CANopen
- [ ] DeltaV
- [ ] DF1
- [ ] EtherNet/IP
- [ ] Firmata
- [ ] KNXnet/IP
- [ ] Modbus
- [ ] OPC-UA
- [X] S7

**Comment by chrisdutz** [comment]
Yikes ... seems I added the reply to this issue to the wrong one ...

I think this might be related to a problem I observed with the serial transport that I wasn't able to configure the settings for the serial transport as it simply wasn't processed. I know that this once worked, but not quite sure what changed. I'll be investigating this once I find some time.

If you however want to give it a try, I'm happy to help.
